pkg/addr/host.go

```diff
@@ -92,6 +92,14 @@ const (
 type HostAddr interface {
 	//@ pred Mem()
 
+	// NOTE[henri]: I rewrote this comment to be more accurate
+	// Return whether all the underlying data that needs to be low for the
+	// computation of `Pack`, `IP`, `String` is low.
+	//@ ghost
+	//@ requires Mem()
+	//@ decreases
+	//@ pure IsLow() bool
```

Comment:
this should already be marked as a hyperfunction to guarantee our assumptions make sense